commit 86724e40bfa544a5024a2a3d522934aef6914cc7
Author: Sterling Augustine <saugustine@google.com>
Date:   Wed Mar 20 17:37:23 2019 +0000

    Make __cpu_model a hidden symbol, to match libgcc.
    Also hide __cpu_inicator_init and __cpu_features2
    for similar reasons.
    
    Summary: Make __cpu_model a hidden symbol, to match libgcc.
    
    Subscribers: llvm-commits
    
    Tags: #llvm
    
    Differential Revision: https://reviews.llvm.org/D59561
    
    llvm-svn: 356581

diff --git a/compiler-rt/lib/builtins/cpu_model.c b/compiler-rt/lib/builtins/cpu_model.c
index fc578203097..5bcc386f4cd 100644
--- a/compiler-rt/lib/builtins/cpu_model.c
+++ b/compiler-rt/lib/builtins/cpu_model.c
@@ -592,14 +592,24 @@ static void getAvailableFeatures(unsigned ECX, unsigned EDX, unsigned MaxLeaf,
 #define CONSTRUCTOR_ATTRIBUTE
 #endif
 
+#ifndef _WIN32
+__attribute__((visibility("hidden")))
+#endif
 int __cpu_indicator_init(void) CONSTRUCTOR_ATTRIBUTE;
 
+#ifndef _WIN32
+__attribute__((visibility("hidden")))
+#endif
 struct __processor_model {
   unsigned int __cpu_vendor;
   unsigned int __cpu_type;
   unsigned int __cpu_subtype;
   unsigned int __cpu_features[1];
 } __cpu_model = {0, 0, 0, {0}};
+
+#ifndef _WIN32
+__attribute__((visibility("hidden")))
+#endif
 unsigned int __cpu_features2;
 
 /* A constructor function that is sets __cpu_model and __cpu_features2 with
